src/Loggers/LogCustomScalar.jl

```diff
@@ -0,0 +1,54 @@
+"""
+Groups multiple scalars in the same plot to be visualized by the CUSTOM_SCALARS
+plugin. Note that this function sets the metadata: the actual values must be
+logged separately with `log_value` and referenced with the correct tag.
+
+The `layout` argument is structured as follows:
+
+layout = Dict(category => Dict(name => (chart_type, [tag1, tag2, ...])))
+
+where `category` is the main tag for the plot, `name` is the plot's name,
+`chart_type` is one between "Multiline" and "Margin" and the array of tags
+contains the actual references to the logged scalars.
+"""
+function log_custom_scalar(logger::TBLogger, layout::AbstractDict; step = nothing)
+    summ = SummaryCollection(custom_scalar_summary(layout))
+    write_event(logger.file, make_event(logger, summ, step = step))
+end
+
+function chart(name::String, metadata::Tuple{AbstractString, AbstractArray})
+    chart_type, tags = metadata
+
+    if chart_type == "Multiline"
```

Comment:
Could you create an enum
```Julia
@enum tb_chart_type tb_multiline=1 tb_margin=2
```
It's less likely to cause errors and easier to work with

Reply:
also export the 2 possible values in `TensorBoard.jl`